bib-image.sh

```diff
@@ -1,6 +1,9 @@
 #!/bin/bash
 set -exuo pipefail
 
+sudo podman login -u "${QUAY_USERNAME}" -p "${QUAY_PASSWORD}" quay.io
+podman login -u "${QUAY_USERNAME}" -p "${QUAY_PASSWORD}" quay.io
```

Comment:
I am curious about requiring both root and user auth here.  Seems like it could use a comment?

Reply:
Because `skopeo inspect` in `shared_lib.sh` use user auth, but for bib test, the follow-up podman bib cmd need root auth. I know this is ugly, but haven't find a good solution yet.